src/main/java/frc/robot/commands/RotateVelocityDriveCommand.java

```diff
@@ -20,18 +22,24 @@ public class RotateVelocityDriveCommand extends CommandBase {
   private final DoubleSupplier translationYSupplier;
   private final DoubleSupplier rotationSupplier;
 
+  private final BooleanSupplier isRobotRelativeSupplier;
+
   /** Creates a new RotateVelocityDriveCommand. */
   public RotateVelocityDriveCommand(
       DrivebaseSubsystem drivebaseSubsystem,
       DoubleSupplier translationXSupplier,
       DoubleSupplier translationYSupplier,
-      DoubleSupplier rotationSupplier) {
+      DoubleSupplier rotationSupplier,
+      BooleanSupplier isRobotRelativeSupplier) {
 
     this.drivebaseSubsystem = drivebaseSubsystem;
     this.translationXSupplier = translationXSupplier;
     this.translationYSupplier = translationYSupplier;
     this.rotationSupplier = rotationSupplier;
+    this.isRobotRelativeSupplier = isRobotRelativeSupplier;
 
+    Shuffleboard.getTab("Drivebase")
+        .addBoolean("Is robot relative drive?", isRobotRelativeSupplier);
```

Comment:
if this is just for testing, can we comment it out before we merge it? each shuffleboard value adds network traffic and cpu load.